docs/v2/5_erc721-interactions.md

```diff
@@ -20,7 +20,7 @@ Here's a video explanation to help you better understand our new ERC-721 prefabs
 
 :::info
 
-In the following code snippet examples, we will use an ERC-721 token contract, as found on the Goerli testnet, called "Chain721 (C721)" for demonstration purposes. We use "0xd25b827D92b0fd656A1c829933e9b0b836d5C3e2" as the example address to fetch from.
+In the following code snippet examples, we will use an ERC-721 NFT token contract, as found on the Goerli testnet, called "Chain721 (C721)" for demonstration purposes. We use "0xd25b827D92b0fd656A1c829933e9b0b836d5C3e2" as the example address to fetch from.
```

Comment:
ERC721 and NFT are redundant, just say one or the other or put NFT in () to clarify.

Reply:
will update, thx